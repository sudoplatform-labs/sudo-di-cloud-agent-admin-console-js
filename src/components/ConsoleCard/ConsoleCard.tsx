import { Card } from 'antd';
import styled from 'styled-components';
import { theme } from '../../theme';

const StyledCard = styled(Card)`
  border-radius: 8px;
  box-shadow: 0 2px 12px 0 rgba(0, 0, 0, 0.12);
`;

const Heading = styled.h2``;

const SubHeading = styled.h3`
  font-size: 14px;
  color: ${theme.greys.raven};
  font-weight: normal;
`;

export const ConsoleCard = Object.assign(StyledCard, {
  Heading,
  SubHeading,
});
