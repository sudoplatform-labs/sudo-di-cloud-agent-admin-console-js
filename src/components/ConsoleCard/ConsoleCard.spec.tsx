import { shallow } from 'enzyme';
import React from 'react';
import { ConsoleCard } from './ConsoleCard';

describe('ConsoleCard', () => {
  it('should render correctly', () => {
    const component = shallow(<ConsoleCard />);

    expect(component).toMatchSnapshot();
  });
});
