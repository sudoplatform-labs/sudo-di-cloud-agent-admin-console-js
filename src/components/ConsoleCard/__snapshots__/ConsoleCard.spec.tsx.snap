// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ConsoleCard should render correctly 1`] = `
.c0 {
  border-radius: 8px;
  box-shadow: 0 2px 12px 0 rgba(0,0,0,0.12);
}

<Card
  className="c0"
/>
`;
